src/interfaces/StatusEnum.tsx

```diff
@@ -0,0 +1,5 @@
+export enum StatusEnum {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
```

Comment:
it is more common to have a single file for types or interfaces per some limited functionality set.
Also, mind the extension - the common convention is to have `tsx` / `jsx` extensions only for files, that contain JSX